content/cloud-docs/run/modes-and-options.mdx

```diff
@@ -47,6 +47,22 @@ The [replace option](/cli/commands/plan#replace-address) instructs Terraform to
 - **CLI:** Use `terraform plan -replace=ADDRESS` or `terraform apply -replace=ADDRESS`.
 - **API:** Use the `replace-addrs` option.
 
+## Allow Empty Apply
+
+A no-operation (empty) apply enables Terraform Cloud to apply a run that produces no changes rather than simply finishing after the plan stage. Empty applies are particularly useful for users wishing to upgrade a Terraform Cloud workspace's latest state version to the desired Terraform version.
+
+-> **Note:** Terraform Cloud cannot guarantee that a plan in this mode will produce no changes and so it is advised to check for resource drift beforehand.
+
+- **API:** Set the `allow-empty-apply` field to `true`.
+- **UI:** Use the "Start new plan" action from a workspace's "Actions" menu, then select the "Allow empty apply" option in the new plan dialog.
+
+## Plan Only
```

Comment:
I think we need to specify in this section that this is a speculative plan. Maybe we need to change the title to "Plan Only - Speculative Plan" or something similar. 

then we need to link to this new section you're making from this existing section on speculative plans: https://www.terraform.io/cloud-docs/run#speculative-plans. I recommend doing an in-text link. Something like : "Speculative plans are [Plan Only runs](/cloud-docs/run/modes-and-options). They show a set of possible changes and check them against Sentinel policies, but cannot apply those changes."

My concern here is just. We need to make sure that people understand that Plan Only = Speculative Plan. Thoughts?

Reply:
Great idea. I was thinking that we should just stick with one phrase but upon further inspection Speculative plans are already documented extensively so it makes sense to connect these together.